### DISCUSSION TITLE
无法编译c++客户端

### DISCUSSION THREAD

**Discussion by macrossgithub** [apache/iotdb#7699] [Q&A]
在编译客户端和客户端应用的时候，会出现如下的错误。
貌似，thrift的编译错误导致无法继续。

有没有经历过的朋友，分享一下。
我的环境是 ubuntu18.04， gcc-9 ， g++-7 。

![2022-10-23_193211](https://user-images.githubusercontent.com/3234867/197389712-29e1e3b6-7a1f-443c-bb55-468f33efd9a7.png)

**Comment by Pummole** [comment]
解决了嘛？我的是到clien-cpp那一步FAILURE
